Code/GraphMol/FileParsers/PDBParser.cpp

```diff
@@ -449,6 +449,23 @@ static void StandardPDBResidueChirality(RWMol *mol) {
   }
 }
 
+void BasicPDBCleanup(RWMol &mol) {
+  ROMol::VERTEX_ITER atBegin, atEnd;
+  boost::tie(atBegin, atEnd) = mol.getVertices();
```

Comment:
The `VERTEX_ITER` approach is a bit more direct (the atom iterators do the same thing under the covers).